packages/@webex/plugin-meetings/src/mediaQualityMetrics/config.ts

```diff
@@ -165,7 +165,7 @@ export const emptyVideoReceiveStream = {
     ssci: 0, // Not avaliable
   },
   h264CodecProfile: 'BP',
-  isActiveSpeaker: true,
+  isActiveSpeaker: false,
```

Comment:
So far, from my understanding, we need only set `isActiveSpeaker` to `true` when the person on a call is talking, and we see this active video from the user, right? Or did I miss something?

@edvujic @SomeBody16

Reply:
Or if the person was active/ speaking for the last 60 sec?

Reply:
@szotrabh Yes, so my point of view is that the user initializes with `false`. When a person is an active speaker, we need to be able to send out information that this user was the active speaker within the last 60 seconds. There is logic involved in calculating the time for that.